gax/retry_loop.h

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Google Inc.  All rights reserved
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "gax/backoff_policy.h"
+#include "gax/call_context.h"
+#include "gax/retry_policy.h"
+#include "gax/status.h"
+#include <chrono>
+#include <functional>
+#include <thread>
+#include <type_traits>
```

Comment:
Do you use this?

Reply:
The `type_traits` include is a leftover from when I tried to pass the next stub call as a functor.